docs/examples/echo-bot-module.js

```diff
@@ -31,8 +31,8 @@ bot.help(async (ctx) => {
   const info = commands.reduce((acc, val) => `${acc}/${val.command} - ${val.description}\n`, '')
   return ctx.reply(info)
 })
-bot.action('delete', ({ deleteMessage }) => deleteMessage())
+bot.action('delete', (ctx) => ctx.deleteMessage())
 bot.on('dice', (ctx) => ctx.reply(`Value: ${ctx.message.dice.value}`))
-bot.on('message', (ctx) => ctx.telegram.sendCopy(ctx.chat.id, ctx.message, keyboard))
+bot.on('message', (ctx) => ctx.telegram.sendCopy(ctx.message.chat.id, ctx.message, keyboard))
```

Comment:
Should we switch over to `copyMessage` here and at all other places? It does require one extra argument, but `sendCopy` is deprecated.

Reply:
Probably, but I can't be bothered.

Reply:
Not exactly in scope of this PR anyway. We can do it while revamping docs, or even later. Not required before 4.0 either.